checker/src/main/java/org/checkerframework/checker/mustcall/messages.properties

```diff
@@ -1,3 +1,5 @@
 inconsistent.mustcall.subtype=%s is annotated as %s, but one of its supertypes is annotated %s
 createsmustcallfor.target.unparseable=The method %s is annotated as @CreatesMustCallFor, but the target (%s) was unparseable in the current context. Rewrite your code so that the relevant expression is a local variable.
 mustcall.not.inheritable=@MustCall annotation is not inherited by subclasses. Use @InheritableMustCall instead.
+mustcallalias.method.return.and.param=A @MustCallAlias annotation must appear on a return type and a parameter type
```

Comment:
Let's add a string argument to this error message that indicates which location is missing an annotation. I suggest the following: 
```@MustCallAlias annotations must appear in pairs (one on a return type and one a parameter type), but %s```, with the following completions for the `%s`:
* `this method has a void return.`
* `there is no @MustCallAlias annotation on the return type, even though parameter $p is annotated as @MustCallAlias.`
* `no parameter has an @MustCallAlias annotation, even though the return type is @MustCallAlias.`

You'll have to distinguish between the last two in the code that calls `checker.reportError`, but I think that's worthwhile if it leads to more actionable error messages.